### PR TITLE
Revives CMB Black Market Investigation

### PR DESCRIPTION
**Pull request by realforest2001**
# Revival of #3223 by BeagleGaming1
# About the pull request

Ordering black market goods can either lower or increase a number. At 100, the CMB gets called to investigate and disable the black market.

# Explain why it's good for the game

People can purchase unlimited goods such as weapons or CLF crates. This causes req to have to at least pay attention to what they are buying, and to buy stuff other than clothing or weapons from other factions.

# Changelog

:cl: BeagleGaming1, forest2001
add: Adds a CMB ERT to investigate the Black Market
add: Adds a [cmb] logo for papers
add: Adds a quiet_launch var for distress beacons, to differentiate between ERTs who should announce their response reception, but not the beacon going out in the first place.
del: Removes banhammer from normal acquisition.
/:cl:

### CODE REVIEW

**Review by sleepynecrons — CHANGES_REQUESTED** [review]
scanner is missing the scanner_on animation the original PR had 
scanr

**Review by sleepynecrons — APPROVED** [review]
sprites ✅

File: code/datums/emergency_calls/inspection.dm [hunk 1]
```diff
@@ -263,3 +263,45 @@
...
+	max_synths = 0
+
+/datum/emergency_call/inspection_cmb/black_market/New()
+	..()
+	dispatch_message = "Third Fleet High Command to [MAIN_SHIP_NAME], we have received inconsistent supply manifests and irregularities on the ASRS system aboard your ship, and have requested a CMB Investigation Team to board and clear you of any wrongdoing."
+	arrival_message = "Incoming Transmission: [MAIN_SHIP_NAME], this is Anchorpoint Station with the Colonial Marshal Bureau. Be advised, we are dispatching a team of Marshals to board with you by request of GSO-91. Submitting authorized docking clearances, over."
+	objectives = "Investigate the inconsistencies aboard the [MAIN_SHIP_NAME]'s ASRS. In the case of illegal activity, collect evidence, and submit a report to the CMB Command at Anchorpoint Station. If required, the ICC Liaison's Tradeband is capable of fixing ASRS computers. Work with the [MAIN_SHIP_NAME]'s military police force."
+
+	will_spawn_icc_liaison = TRUE
+	will_spawn_cmb_observer = FALSE
```
Comment (harryob) [inline code/datums/emergency_calls/inspection.dm:282]: why are these being set in new they're constants

File: code/modules/gear_presets/cmb.dm [hunk 2]
```diff
@@ -344,6 +344,14 @@
...
 	new_human.equip_to_slot_or_del(new /obj/item/tool/crowbar/red, WEAR_IN_BACK)
 
+/datum/equipment_preset/cmb/liaison/black_market
+	name = "CMB - ICC Liaison - Black Market ERT"
+	skills = /datum/skills/civilian/icc_investigation
+
+/datum/equipment_preset/cmb/liaison/black_market/load_gear(mob/living/carbon/human/H)
+	. = ..()
+	H.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full, WEAR_WAIST) //Tool belt to open ASRS
+	H.equip_to_slot_or_del(new /obj/item/device/cmb_black_market_tradeband, WEAR_IN_BACK) //Tradeband to disable black market
```
Comment (harryob) [inline code/modules/gear_presets/cmb.dm:354]:
```suggestion
/datum/equipment_preset/cmb/liaison/black_market/load_gear(mob/living/carbon/human/new_human)
	. = ..()
	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full, WEAR_WAIST) //Tool belt to open ASRS
	new_human.equip_to_slot_or_del(new /obj/item/device/cmb_black_market_tradeband, WEAR_IN_BACK) //Tradeband to disable black market
```

File: maps/map_files/Ice_Colony_v3/Shivas_Snowball.dmm
Comment (harryob) [inline maps/map_files/Ice_Colony_v3/Shivas_Snowball.dmm:1]: undocumented

File: code/game/objects/effects/spawners/random.dm
Comment (harryob) [inline code/game/objects/effects/spawners/random.dm:1]: undocumented

File: code/datums/supply_packs/black_market.dm [hunk 3]
```diff
@@ -622,7 +631,6 @@ Primarily made up of things that would be best utilized, well, shipside. Recreat
 		/obj/item/explosive/grenade/smokebomb,
 		/obj/item/corncob,
 		/obj/item/poster,
-		/obj/item/weapon/banhammer,
```
Comment (harryob) [inline code/datums/supply_packs/black_market.dm:625]: undocumented

File: code/game/objects/items/circuitboards/computer.dm [hunk 4]
```diff
@@ -178,11 +185,39 @@
...
 				if(!contraband_enabled)
-					to_chat(user, SPAN_WARNING("You amplify the broadcasting function with \the [multitool], and a red light starts blinking on and off on the board. Put it back in?"))
+					to_chat(user, SPAN_WARNING("You amplify the broadcasting function with \the [tool], and a red light starts blinking on and off on the board. Put it back in?"))
 					contraband_enabled = TRUE
 				else
-					to_chat(user, SPAN_WARNING("You weaken the broadcasting function with \the [multitool], and the red light stops blinking, turning off. It's probably good now."))
+					to_chat(user, SPAN_WARNING("You weaken the broadcasting function with \the [tool], and the red light stops blinking, turning off. It's probably good now."))
 					contraband_enabled = FALSE
+
+	if(istype(tool, /obj/item/device/cmb_black_market_tradeband))
```
Comment (harryob) [inline code/game/objects/items/circuitboards/computer.dm:194]: trait check

File: code/game/supplyshuttle.dm [hunk 5]
```diff
@@ -1234,6 +1250,11 @@ var/datum/controller/supply/supply_controller = new()
 	/// For code readability.
 	addtimer(CALLBACK(GLOBAL_PROC, /proc/playsound, get_rand_sound_tile(), sound_to_play, 25, FALSE), timer)
 
+/datum/controller/supply/proc/black_market_CMB_investigation()
```
Comment (harryob) [inline code/game/supplyshuttle.dm:1253]: all paths should be snake case including initialisms

**Review by harryob — CHANGES_REQUESTED** [review]

File: maps/map_files/Ice_Colony_v3/Shivas_Snowball.dmm
Comment (realforest2001) [reply maps/map_files/Ice_Colony_v3/Shivas_Snowball.dmm:1]: *pain
I put it in the changelog and didn't save it

**Review by harryob — APPROVED** [review]

**Comment by kooarbiter** [comment]
time to call in enough black market weapons that a CMB team is codewise forced to arrive, conveniently before the aliens hijack so they have to help out, big brained moment for free ert

**Pull request merged**